CODEOWNERS

```diff
@@ -109,6 +109,8 @@
 
 /Workbooks/Azure*Monitor*-*Workspaces/** @microsoft/azure-monitor-workspaces
 
+/Workbooks/Advisor/** @microsoft/advisor-waf-sme-workbooks
```

Comment:
Let's make it Azure Advisor. I believe it will look like this: /Workbooks/Azure Advisor/** @microsoft/advisor-waf-sme-workbooks